### DISCUSSION TITLE
How to include multiple license files

### DISCUSSION THREAD

**Discussion by gee4vee** [microsoft/vscode-discussions#399] [Extension Development QnA] [answered]
The license for our extension is translated into many languages in separate files. Is there a built-in way to include multiple license files in the published extension? If so how do we specify them in the manifest?

**Comment by isidorn** [comment, marked as answer]
Currently this is not supported. You can only have one license in your extension.
I am not sure is it ok from a legal perspective, but maybe you could append all of them in one file?
In the future there might be some support for localisation by the marketplace, but this is currently not supported.

**Comment by gee4vee** [comment]
@isidorn thanks for responding! Should I open a feature request for this?

**Reply by isidorn** [reply]
Yes. A good place would be this repo https://github.com/microsoft/vsmarketplace

**Reply by gee4vee** [reply]
https://github.com/microsoft/vsmarketplace/issues/556
